config/meta.yml

```diff
@@ -0,0 +1,5 @@
+meta_product_name: "DaybyDay"
+meta_title: "DaybyDay - Take Action with AI Journaling App"
```

Comment:
could you please insert "the":

DaybyDay - Take Action with **the** AI Journaling App